src/main/target/BEEBRAIN_V2F/target.h

```diff
@@ -83,9 +83,9 @@
 #define USE_VTX_RTC6705
 #define USE_VTX_RTC6705_SOFTSPI
 #define USE_VTX_CONTROL
-#define RTC6705_SPI_MOSI_PIN    PC15
+#define RTC6705_SPIDATA_PIN     PC15
 #define RTC6705_SPICLK_PIN      PC13
-#define RTC6705_CS_PIN          PB12
+#define RTC6705_SPILE_PIN       PB12
```

Comment:
Why are you changing these back? These were renamed for consistency in #4918?

Reply:
ahh i did wonder what happened there but missed that PR.

However, the RTC6705 does is NOT actually a true SPI bus device, I find it misleading to have defines that make people think it is just like any other SPI device.

Having the names as they are in this PR also makes it easier for people to 
a) design schematics using them and b) wire one up
c) cross reference the board silk screens on the NEO VTX board + SPRacingF7DUAL VTX board with the code.

Reply:
this change is also in the other PR for VTX power control. #5279

Reply:
I (and @martinbudden) found it to be misleading to have different names for pins with (essentially) the same functions between soft SPI and SPI. Not sure what the right thing is there, but I think board silk screens should not be used to influence the discussion.